packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -855,6 +872,21 @@ export const stripeRoutes = (
       },
       handler: (request: AuthRequest) => stripeHandler.previewInvoice(request),
     },
+    {
+      method: 'POST',
+      path: '/oauth/subscriptions/coupon',
+      options: {
+        auth: false,
+        validate: {
+          payload: {
+            priceId: validators.subscriptionsPlanId.required(),
+            promotionCode: isA.string().required(),
+          },
+        },
```

Comment:
This is missing a response schema for response validation.

Reply:
added